openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OParamUtils.java

```diff
@@ -30,10 +29,10 @@
  * Abstract class to parse H2O algorithm params.
  *
  * @param <T> The concrete type of {@link ModelParametersSchemaV3 algorithm params}.
- * @since 0.1.0
- * @author Pedro Rijo (pedro.rijo@feedzai.com)
+ * @since 1.0.0
```

Comment:
I think that if a class keeps it's name, even if the logic change, the since should be kept. not sure. cc @nmldiegues ?